assets/airports/othh.json

```diff
@@ -11,176 +10,1785 @@
+        "ORMAL": ["N25d23m03.74", "E052d22m00.97"],
+        "ORSIS": ["N25d28m01.00", "E052d16m36.00"],
+        "OTIDA": ["N25d08m03.98", "E051d33m34.88"],
+        "OTINO": ["N25d27.68", "E52d15.43"],
+        "OVENO": ["N25d22m49.23", "E051d29m33.40"],
+        "OVONA": ["N25d24m43.00", "E052d47m39.00"],
+        "PASAV": ["N25d15m55.53", "E051d55m50.50"],
+        "PASID": ["N25d05m15.69", "E051d40m45.07"],
+        "PATIS": ["N25d40.77", "E52d21.69"],
+        "PATOM": ["N25d58m21.41", "E051d18m36.36"],
+        "PEGEM": ["N25d19m25.12", "E052d06m39.78"],
+        "PURIN": ["N25d03m41.15", "E051d33m14.60"],
+        "PURLI": ["N25d36.73", "E53d24.60"],
+        "PUSNO": ["N25d17m58.69", "E051d55m12.34"],
+        "PUTAN": ["N25d27m54.71", "E051d38m00.57"],
+        "PUXEL": ["N25d04m23.94", "E051d35m11.76"],
+        "RABLA": ["N26d15.09", "E51d48.56"],
+        "RAMKI": ["N26d11.63", "E51d56.41"],
+        "RASBO": ["N24d33.70", "E51d28.28"],
+        "RASDI": ["N26d04m25.38", "E051d24m07.37"],
+        "RESAR": ["N25d37m07.00", "E052d23m28.00"],
+        "ROSAN": ["N26d31.49", "E51d52.33"],
+        "RUGIS": ["N24d59.27", "E53d3.67"],
+        "SALWA": ["N25d15m38.00", "E050d30m48.00"],
+        "SIDMU": ["N25d11m31.44", "E051d45m13.15"],
+        "SITAL": ["N25d35m10.20", "E052d09m27.90"],
+        "SI"   : ["N25d53.66", "E50d36.22"],
+        "SODER": ["N24d59m40.88", "E051d32m50.24"],
+        "SOGAN": ["N26d39.25", "E51d54.13"],
+        "SOKEN": ["N25d27m04.81", "E051d32m25.39"],
+        "SOLAL": ["N24d45m58.18", "E051d34m54.33"],
+        "SOMAL": ["N23d28.73", "E51d27.27"],
+        "SOMAP": ["N25d06m13.80", "E051d32m07.37"],
+        "TABKI": ["N25d38m30.20", "E051d50m35.18"],
+        "TANDO": ["N25d28m53.68", "E051d24m22.68"],
+        "TANLA": ["N25d40.28", "E51d54.10"],
+        "TONBI": ["N24d41m04.68", "E051d33m44.82"],
+        "TORKO": ["N25d10m40.17", "E051d39m39.39"],
+        "TOSNA": ["N25d16m12.00", "E052d41m16.00"],
+        "TOSTA": ["N26d27.77", "E50d49.21"],
+        "TOTIS": ["N26d11m19.11", "E051d10m26.90"],
+        "TOVAN": ["N25d31m09.81", "E051d30m37.05"],
+        "TUKEN": ["N25d36m50.90", "E051d31m16.64"],
+        "UKILA": ["N25d34m26.51", "E051d49m47.60"],
+        "UKILI": ["N24d38.26", "E53d56.61"],
+        "UKPUX": ["N24d36m10.41", "E051d32m35.22"],
+        "ULBEK": ["N25d27m25.61", "E052d16m09.50"],
+        "ULELO": ["N25d35m43.81", "E052d01m27.13"],
+        "ULIKA": ["N25d15m45.29", "E050d38m48.79"],
+        "UMIBU": ["N24d23.52", "E54d30.45"],
+        "VATEK": ["N25d47m26.68", "E051d43m06.54"],
+        "VEDED": ["N26d05m58.26", "E051d46m27.90"],
+        "VEKEL": ["N26d19.48", "E53d57.63"],
+        "VELAM": ["N25d54m26.12", "E051d43m47.13"],
+        "VUVAN": ["N24d54m15.69", "E051d44m45.27"],
+        "WK"   : ["N25d10.88", "E51d36.00"],
+        "XABAB": ["N25d06m41.77", "E051d58m23.88"],
+        "XALTO": ["N24d59m24.78", "E052d03m30.62"]
     },
-
+    "restricted": [
```

Comment:
I'd normally say not to do this, but could you add the danger areas as well. I think they would look good, especially since some procedures are designed around them

Reply:
That would impair the GINTO arrivals.
![Screenshot](https://i.imgur.com/l01NwtT.png)

Reply:
In that case, just disregard what I said. Let's stick with only the restricted areas